Ambassador Bylaws/Article2.html

```diff
@@ -224,6 +229,10 @@ <h2 id="article2">Article 2. Roles of Ambassadors</h2>
       publish updated scores and an updated vesting schedule for each Ambassador noting their current
       allocation and vesting. The score for any Ambassador that did not provide Submissions will be 0 for
       that period.</li>
+    <li>Ambassadors are expected to demonstrate meaningful contribution each month toward the responsibilities
+      of their selected Primary Team, as outlined in Article 3, Section 1(c).</li>
```

Comment:
The way the <ol> gets rendered by github pages means this reference should be updated from 1(c) to 1.3